### PR TITLE
docs(md): improve code blocks

### PR DESCRIPTION
**Pull request by ctison**
### Summary

Embellish documentation with syntax coloring.

In markdown code blocks:
- Add the missing space between the language and the line highlights[^1], eg:
```diff
-```toml{4}
+```toml {4}
```
- Set the code block `title`[^2] from the first line if it is a comment looking like a file path, eg:
```diff
-```js
-// web/src/index.js
-
+```js title="web/src/index.js"
```
- Replace languages:
  - `gql` -> `graphql`
  - `js` and `javascript` -> `jsx`
  - `ts` and `typescript` -> `tsx`
  - `prisma` -> `graphql`
  - `terminal` -> `bash`

[^1]: https://docusaurus.io/docs/markdown-features/code-blocks#line-highlighting
[^2]: https://docusaurus.io/docs/markdown-features/code-blocks#code-title

### CODE REVIEW

**Comment by thedavidprice** [comment]
Thanks @ctison I'm looping in @jtoar to take a look at this. He's our DocsMaster9000™

**Comment by jtoar** [comment]
@ctison Wow this is epic! Thank you so much. I'm going to add a few tweaks for line highlights, but otherwise this is looking great.

File: docs/docs/app-configuration-redwood-toml.md [hunk 1]
```diff
@@ -147,8 +147,7 @@ Examples include: logger settings, timeouts, maximum payload limits, and more.
 
 > **Note:** This configuration does not apply in a serverless deploy.
 
-```js
-// ./api/server.config.js
+```jsx title="./api/server.config.js"
```
Comment (jtoar) [inline docs/docs/app-configuration-redwood-toml.md:150]:
```suggestion
```jsx title="api/server.config.js"
```

Comment (jtoar) [inline docs/docs/app-configuration-redwood-toml.md:150]:
```suggestion
```javascript title="api/server.config.js"
```

File: docs/docs/webpack-configuration.md [hunk 2]
```diff
@@ -19,9 +19,7 @@ yarn rw setup webpack
 
 This command adds a file called `webpack.config.js` to your project's `web/config` directory, creating `web/config` if it doesn't exist:
 
-```js
-// web/config/webpack.config.js
-
+```jsx title="web/config/webpack.config.js"
```
Comment (jtoar) [inline docs/docs/webpack-configuration.md:22]:
```suggestion
```javascript title="web/config/webpack.config.js"
```

File: docs/docs/storybook.md [hunk 3]
```diff
@@ -98,9 +94,7 @@ yarn workspace web add -D @storybook/addons @storybook/theming
 
 Then, we'll configure our theme by creating a `storybook.manager.js` file. Below we're enabling Storybook's dark theme.
 
-```js
-// web/config/storybook.manager.js
-
+```jsx title="web/config/storybook.manager.js"
```
Comment (jtoar) [inline docs/docs/storybook.md:97]:
```suggestion
```javascript title="web/config/storybook.manager.js"
```

File: docs/docs/serverless-functions.md [hunk 4]
```diff
@@ -797,9 +784,7 @@ By default, RedwoodJS functions return strings or JSON. If you need to return bi
 
 Here's an example of how to return a binary file from the filesystem:
 
-```typescript
-// api/src/functions/myCustomFunction.ts
-
+```tsx title="api/src/functions/myCustomFunction.ts"
```
Comment (jtoar) [inline docs/docs/serverless-functions.md:787]:
```suggestion
```typescript title="api/src/functions/myCustomFunction.ts"
```

**Pull request merged**